readme.md

```diff
@@ -66,6 +66,10 @@ The number of promises that are currently running.
 
 The number of promises that are waiting to run (i.e. their internal `fn` was not called yet).
 
+### limit.clearQueue()
+
+Discards any pending promises that are waiting to run.
```

Comment:
Could you also mention one or more use-cases for this? For example, the use-case mentioned in the issue. It's good to mention use-cases as it's not always immediately clear to a user whether they would need something or not.

Reply:
Sorry for the delay, I've added a couple use cases where I think this would be useful (1 of which is the aforementioned case from the FR). PTAL!